src/stories/containers/TransparencyReport/components/HeaderWithIcon/TooltipHeader.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from '@emotion/styled';
+import { CustomLink } from '@ses/components/CustomLink/CustomLink';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import React from 'react';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+import type { CSSProperties } from 'react';
+
+interface Props {
+  description: string;
+  mipNumber: string;
+  link: string;
+  name: string;
+  style?: CSSProperties;
+}
+
+const HeaderToolTip: React.FC<Props> = ({ description, mipNumber, link, style, name }) => {
+  const { isLight } = useThemeContext();
+  return (
+    <Container style={style}>
+      <Description isLight={isLight}>{description}</Description>
+      <Source isLight={isLight}>Source</Source>
+      <ContainerLinkWithMip isLight={isLight}>
+        <MipNumber>{mipNumber}</MipNumber>
+        <ContainerLink>
+          <CustomLink
+            children={name}
```

Comment:
use the notation `<CustomLink>{name}</CustomLink>` instead of using the children prop

Reply:
done